src/lib/moo-db.js

```diff
@@ -39,7 +40,16 @@ class MooDB {
   load(dir = '') {
     const ids = this.fsdb.lsDirs(dir);
     ids.forEach(id => {
-      this.loadObject(id.replace(/[\/\\]/g, '_'));
+      const filepath = path.posix.join(dir, id);
+      // Recurse
+      this.load(filepath);
+
+      // Load object contents
+      // Not the most efficient way to check that the directory
+      // has content to load, but we only do that at start-up.
+      if (this.fsdb.lsFiles(filepath).length) {
```

Comment:
Maybe you can add a isEmpty(filepath) or hasContent(filepath) method to the fsdb.